### DISCUSSION TITLE
Weasel.Postgresql Version

### DISCUSSION THREAD

**Discussion by stavros-georgiou-cy** [JasperFx/marten#2059] [Q&A]
Hello, We are using Marten version 4.3.1 with .NET 5 and we want to migrate to .NET 6. We found that in version 4.3.1 the Weasel.Postgresql is 1.3.0. 

Is it fine to upgrade to .NET 6 even if Weasel.Postgresql is not the latest? (3.0.0) 

**Edit 1:** After a research we found that we cannot use `Microsoft.EntityFrameworkCore 6.0.2`.

Thank you.

**Comment by jeremydmiller** [comment]
There's a Marten 5.0.0-alpha-* release that targets .Net 6 with updated Weasel and Npgsql dependencies. No big changes from v4 (but some are coming later), just enabling .Net 6.

**Reply by stavros-georgiou-cy** [reply]
The issue now is that we are having 2 DBs; one for write model (marten) and one for read model (another postgres). We want to upgrade to the latest version of EntityFramework for the read model but we can't. 

So your proposal is to upgrade to the alpha version? or just wait with the migration of the EntityFramework until version 5. 

I found that .NET is fine to upgraded to version 6 but keep the EntityFramework to 5.0.x. Am i right or i miss something here? 

Thank you again! :)
